### DISCUSSION TITLE
Difference between torch.manual_seed and torch.random.manual_seed

### DISCUSSION THREAD

**Discussion by Durga-33** [mrdbourke/pytorch-deep-learning#246] [Q&A]
Kindly explain difference between 
torch.manual_seed(seed=RANDOM_SEED) and 
torch.random.manual_seed(seed=RANDOM_SEED)

This is given in screenshot below
<img width="438" alt="image" src="https://user-images.githubusercontent.com/122006734/210726114-70023ea1-4fca-4155-ac40-c9a97f13f95f.png">
in 
00. PyTorch Fundamentals

**Comment by tanmaypandey7** [comment]
I checked the doc, and looks like their source point to the same link, so it should do the same thing

[`torch.manual_seed`](https://pytorch.org/docs/stable/generated/torch.manual_seed.html#torch.manual_seed) - [Source](https://pytorch.org/docs/stable/_modules/torch/random.html#manual_seed)

[`torch.random.manual_seed`](https://pytorch.org/docs/stable/random.html#torch.random.manual_seed) - [Source](https://pytorch.org/docs/stable/_modules/torch/random.html#manual_seed)

**Reply by Durga-33** [reply]
Thanks
